Makefile

```diff
@@ -252,12 +249,11 @@ CI_GOLANG_DOCKER_MAKE := $(DOCKER) run \
 	-e WASM_ENABLED=$(WASM_ENABLED) \
 	-e FUZZ_TIME=$(FUZZ_TIME) \
 	-e TELEMETRY_URL=$(TELEMETRY_URL) \
-	$(GOIMAGE) \
-	make
+	golang:$(GOVERSION)
 
 .PHONY: ci-go-%
 ci-go-%: generate
-	$(CI_GOLANG_DOCKER_MAKE) $*
+	$(CI_GOLANG_DOCKER_MAKE) /bin/bash -c "git config --system --add safe.directory /src && make $*"
 
```

Comment:
This seems to resolve the git ownership mismatch issue which caused build failures. Not sure if this the right way though.

Reply:
I looks like we could hide the same extra call in `CI_GOLANG_DOCKER_MAKE`, but it's got to happen somewhere.

Reply:
This change is more or less exactly what I'd have recommended to work around the new Git safe/unsafe directory check behavior. Looks good to me.